services/tests/test_flake_detector.py

```diff
@@ -161,32 +161,6 @@ def test_flake_consecutive_differing_outcomes_no_main_branch_specified(dbsession
     assert flaky_tests[test_id] == {FlakeSymptomType.CONSECUTIVE_DIFF_OUTCOMES}
 
 
-def test_flake_consecutive_differing_outcomes_no_main_branch_specified(dbsession):
```

Comment:
@joseph-sentry This test looked to be a copy of the one above it, let me know if we actually wanted to keep

Reply:
if it's an exact copy, it's probably a slip up and can be deleted, but if they differ a bit, then it's probably a name mixup and we should rename and keep

Reply:
Yeah it was an exact copy :o